### ISSUE TITLE
Wiki Table of Contents

### ISSUE THREAD

**Issue by tylerjereddy** [MDAnalysis/mdanalysis#237]
The currently-empty [TableOfContents](https://github.com/MDAnalysis/mdanalysis/wiki/TableOfContents) page on our github wiki should probably be populated with links to match the format we had for the Google Code [TableOfContents](https://code.google.com/p/mdanalysis/wiki/TableOfContents) wiki page.

**Comment by dotsdl** [comment]
Is a [TableOfContents](https://github.com/MDAnalysis/mdanalysis/wiki/TableOfContents) page really necessary? I think all of the major headings in that page would make sense as individual pages, and these could be linked in the sidebar that's visible from every page already. This leaves fewer things to maintain, in my opinion, and is less redundant.

**Comment by dotsdl** [comment]
I went ahead and did what I suggested for the [Release Notes](https://github.com/MDAnalysis/mdanalysis/wiki/Release-Notes). The page probably doesn't need much else, and it pulls in all the release notes to one navigable spot. I also changed the link to the 0.9.1 release notes to point there instead.

**Comment by tylerjereddy** [comment]
Ok, I've removed the TableOfContents page from the wiki (and the link to it from the sidebar). 

I created a new Guide for Developers page (which contains useful links), and then linked to that page from the sidebar. That way all that information is accessible from the sidebar without causing too much clutter. I think that was the last major set of links missing from the github wiki sidebar.

Of course, some of the pages to which these new links point need updating, but that can be done with separate issues as needed.
